### DISCUSSION TITLE
Notification pop up when requesting a "Play Next" song.

### DISCUSSION THREAD

**Discussion by Nxught** [ciderapp/Cider#1411] [Feature Request]
I would like there to be a notification pop up like how you would get on Apple Music on iPhones for instance, when you request a Play next song, or add a song to the library. You would get this temporary UI in the middle of the applications screen stating that, the track you've just requested would be played next. Of course, the content in the UI would have different content depending what option you've chose from the submenu.

Hopefully that made sense.

**Comment by AW1534** [comment]
In the middle of the screen? nah. Apple music did this wrong. It should be in an unobtrusive but obvious place
